src/Features/Lsif/Generator/Generator.cs

```diff
@@ -1,6 +1,11 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
+<<<<<<< HEAD
+=======
+
+#nullable enable annotations
+>>>>>>> 831d9963bbe... Manual fixup
```

Comment:
There are differences with conflict markers in this commit ("manual fixup"). I wasn't able to confirm those were removed in the other commit and I wouldn't expect the automation to have removed them. What's happening?

Reply:
I did a rebase before re-running the generator and it looks like not all my merges got cleaned up. Fixing.